src/proto/Cosmos.proto

```diff
@@ -205,6 +208,30 @@ message Message {
         bytes auth_info_bytes = 2; 
     }
 
+    // Grant gives permissions to execute
+    // the provide method with expiration time.
+    message Grant {
+        google.protobuf.Any authorization = 1;
```

Comment:
App won't have access to `cosmos::staking::v1beta1::StakeAuthorization`, or `google.protobuf.Timestamp`

Reply:
I will expose `cosmos::staking::v1beta1::StakeAuthorization`, but from my understanding, a grant can be many different possibilities. So the app will serialize as base64 `cosmos::staking::v1beta1::StakeAuthorization` and use `auth->set_value()` - regarding the timestamp, it's part of the standard library of protobuf, why would we not have access to it?

Reply:
I also noticed that in this proto we never use Any type, and more importantly, not with the types from the internal protobuf.  Here it should be simple types, and the Any type should be built by code.

Reply:
See handling of SignatureInfo.public_key -- it is an Any type in Protobuf/tx.proto, but it is set using PackFrom in ProtobufSerialization.cpp.
The protobufs in src/Cosmos/Protobuf/* and src/proto/Cosmos.proto should be completely independent; src/Cosmos/Protobuf/* should be internal

Reply:
Fixed, thanks!